web/src/main/java/no/nav/foreldrepenger/tilbakekreving/web/app/tjenester/behandling/BehandlingRestTjeneste.java

```diff
@@ -565,8 +568,10 @@ private BehandlingOperasjonerDto lovligeOperasjoner(Behandling b, boolean finnes
         if (b.erSaksbehandlingAvsluttet()) {
             return BehandlingOperasjonerDto.builder(b.getUuid()).build(); // Skal ikke foreta menyvalg lenger
         } else if (BehandlingStatus.FATTER_VEDTAK.equals(b.getStatus())) {
-            boolean tilgokjenning = b.getAnsvarligSaksbehandler() != null &&
-                !b.getAnsvarligSaksbehandler().equalsIgnoreCase(KontekstHolder.getKontekst().getUid());
+            var sisteSaksbehandler = Optional.ofNullable(b.getAnsvarligSaksbehandler())
+                .or(() -> b.getAksjonspunktMedDefinisjonOptional(AksjonspunktDefinisjon.FORESLÅ_VEDTAK).map(BaseEntitet::getEndretAv));
+            boolean tilgokjenning = sisteSaksbehandler.isPresent() &&
```

Comment:
For konsistens bør samme logikk også implementeres i PipRepository.hentInternBehandlingData